guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -51,6 +51,9 @@ endif::[]
 // Upgrading Capsule Server
 include::topics/upgrading_capsule_server.adoc[leveloffset=+2]
 
+// Synchronizing SmartProxy Servers Through Project Web UI
+include::topics/proc_synchronizing-smartproxy-servers-through-project-web-ui.adoc[leveloffset=+2]
```

Comment:
I'd suggest moving this module to `common/modules` and adapt the include path.

Reply:
Applied.